keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -281,5 +309,23 @@ parameters: string:string [6]
 1.  `releasePolicy`: Prescribes outcome of a Deletion and Revoke events.
     - `Delete`:  the bucket and its contents are destroyed
     - `Retain`:  the bucket and its contents are preserved, only the user’s access privileges are terminated
+    - `Reuse` : TBD
+    - `Erase`: TBD
 - `bucketContentName`: (Optional). An admin defined BucketContent representing a Brownfield or Static bucket.  A non-nil value in this field prevents the BucketClass from being used for Greenfield.
-- `parameters`: object store specific key-value pairs passed to the driver.
\ No newline at end of file
+- `parameters`: object store specific key-value pairs passed to the driver.
+
+#### COSIDriver
```

Comment:
COSIDriverRegistration?

Reply:
Also, is this something we want to include or continue the dialogue before committing to an API?

Reply:
I'd say lets keep it in for now (so there's a description in writing) and call it out in the next review mtg.